runners/pc-usbip/README.md

```diff
@@ -0,0 +1,49 @@
+# USB/IP Simulation
+
+This runner allows using USB/IP as a mean to simulate device connection
+to the OS, and should allow faster development of the embedded applications.
+
+Platform and storage implementations are taken from the Trussed tutorial:
+- https://github.com/trussed-dev/trussed-totp-pc-tutorial
+
+Remarks:
+- At the moment FIDO app only (to be extended with Admin and Provision apps);
+- Works with Chromium and pynitrokey (with a patched fido2.hid module);
```

Comment:
Could you please add some details on the required patch?

Reply:
Looks like you just have to replace `raise OSError("failed to write entire packet")` with `pass` in `FileCtapHidConnection.write_packet` in `fido2`’s `hid/base.py`.

https://github.com/Yubico/python-fido2/blob/dd5662ca7b38ce6b30eb6c916c92bc75006424e7/fido2/hid/base.py#L38